packages/amplify-graphql-api-construct/src/amplify-graphql-definition.ts

```diff
@@ -1,32 +1,90 @@
 import * as os from 'os';
 import { SchemaFile } from 'aws-cdk-lib/aws-appsync';
-import { IAmplifyGraphqlDefinition } from './types';
+import { IAmplifyGraphqlDefinition, ModelDataSourceDefinition } from './types';
+import { getModelTypeNames } from './internal';
 
+export const DEFAULT_MODEL_DATA_SOURCE_DEFINITION: ModelDataSourceDefinition = {
+  name: 'defaultDDB',
```

Comment:
Ah I forgot this point. Since it is only a renaming, I will make this change in the follow up PR if no other major changes required for the current one